back/transcendence_django/back_game/game_arena/arena.py

```diff
@@ -197,13 +197,21 @@ def can_be_over(self) -> bool:
 
     def set_status(self, status: GameStatus):
         self.game.set_status(status)
+        logger.info("Arena %s game status: %s", self.id, status)
 
     def did_player_give_up(self, user_id: int) -> bool:
         return self.player_manager.did_player_give_up(user_id)
 
+    def get_winner(self) -> Player | None:
+        return self.player_manager.winner
+
     def is_private(self) -> bool:
         return self.game.is_private
 
+    async def send_update(self, update_dict: dict[str, Any]):
+        if self.game_update_callback is not None:
```

Comment:
Use assertion since game_update_callback MUST always be defined when these functions are called, so that if it's not, we get an AssertionError instead of a silent fail?
```python
assert self.game_update_callback, "Game update callback undefined even after player joined?"
```